designs/cloudstack-multiple-endpoints.md

```diff
@@ -163,6 +177,27 @@ api-url    = http://172.16.0.3:8080/client/api
 ...
 ```
 
+1. `Global`, `AvailabilityZone2`, and `AvailabilityZone3` are entirely arbitrary names that are unnecessary.
+2. If these tags mismatch between the name in the AvailabilityZone and here there will be an error.
+3. If we use validation, then we can prevent mismatched names, but what about modifying this secret in an existing cluster?
+    - Validation doesn't operate on an existing cluster unless the secret is updated through EKS-A.
+    - If an existing cluster wants to update a secret, the entire secret must be modified together. If this results in
```

Comment:
Is there a mechanism you'd propose to support updating secrets?

Reply:
The particular point you've commented on is more about having multiple secrets to isolate them from each other.

Otherwise, I think an update of a secret could be done directly on a cluster, or through EKS-A.

As for mechanisms, if EKS-A was applying a secret and verifying it, then maybe it could also accept updates thereof. How would a user update a secret today? I think currently they'd have to manually apply it.